README.md

```diff
@@ -59,6 +58,12 @@ changes may be written to a config file in a platform-specific location:
   Alternatively, a development build can be performed by running `go install` in
   a locally checked-out repository.
 
+  If you want to easily access dcrctl via `dcrctl` handle on command-line, since
+  `go install` will put **dcrctl** binary in `go env GOPATH` you might need to
```

Comment:
This isn't correct either.  It goes into the `bin` directory of `go env GOPATH`, no the GOPATH directory itself.

Reply:
Right, missed that and mostly wanted to capture the general idea there, your phrasing from below sounds better though.